scripts/get-pr-comments.sh

```
@@ -0,0 +1,731 @@
+    if ! $temp_auth && gh auth status --hostname "$GITHUB_HOST" &>/dev/null; then
+        echo "→ Using existing GitHub CLI authentication"
+    elif ! $temp_auth; then
+        # Method 3: Interactive authentication
+        echo "No GitHub authentication found."
+        echo ""
+        echo "Choose authentication method:"
+        echo "1. Enter Personal Access Token (recommended for automation)"
+        echo "2. Interactive browser login"
+        echo "3. Show token setup help"
+        echo "4. Exit"
+        read -p "Choice (1-4): " choice
```

Shellcheck (0.10.0)

[info] read without -r will mangle backslashes.

(SC2162)